tests/test_metadata.py

```diff
@@ -23,3 +23,26 @@ def test_generate_requirements():
     ]
     generated_metadata = sorted(set(generate_requirements(extras_require)))
```

Comment:
Maybe the set() is no longer needed

Reply:
That's correct, it could be removed after this library change.